ansible/configs/ansible-multitier-infra/sample_vars/configured_tower_ec2.yml

```diff
@@ -0,0 +1,245 @@
+---
+################################################################################
+### OSP Environment variables
+################################################################################
+env_type: ansible-multitier-infra       # Name of config to deploy
+output_dir: /tmp/output_dir             # Writable working scratch directory
+email: "{{ env_type }}@example.com"     # User info for notifications
+guid: guid01                            # Your Global UNIQUE Identifier
+cloud_provider: ec2                     # Which AgnosticD Cloud Provider to use
+ansible_user: ec2-user
+remote_user: ec2-user
+student_user: devops
+student_password: r3dh4t1!
+devops_password_tower: r3dh4t1!
+key_name: ocpkey
+
+# The region to be used, if not specified by -e in the command line
+aws_region: us-east-1
+
+HostedZoneId: Z3IHLWJZOU9SRT
+
+## Networking (AWS)
+subdomain_base_short: "{{ guid }}"
+subdomain_base_suffix: ".example.opentlc.com"
+subdomain_base: "{{ subdomain_base_short }}{{ subdomain_base_suffix }}"
+zone_internal_dns: "{{ guid }}.internal."
+chomped_zone_internal_dns: "{{ guid }}.internal"
+
+osp_cluster_dns_server: ddns01.opentlc.com
+use_dynamic_dns: true
+osp_project_create: true
+
+# Default instance type
+__instance_type: "t2.medium" 
```

Comment:
Same thing here. For the love of all that is good, please can we not do this for vars? Maybe I am missing a new trend. Or maybe not and it was always a bad idea.

Reply:
Yeah... agree with @stencell on this one. Anything in sample vars is part of the interface and the underscore naming is inappropriate here.